metadata-ingestion/src/datahub/ingestion/source/identity/okta.py

```diff
@@ -260,6 +261,7 @@ def create(cls, config_dict, ctx):
     def __init__(self, config: OktaConfig, ctx: PipelineContext):
         super().__init__(ctx)
         self.config = config
+        self.config.okta_domain = remove_protocol(self.config.okta_domain)
```

Comment:
can we do this in the config pydantic model itself?

Also, can you explain why this is necessary now?

Reply:
It is not necessary. I just was making this bugfix when I saw it. Currently if you pass full URL `https://abc.okta.com` it just fails. It needs to be passed `abc.okta.com` without the protocol which seems odd. I fixed similar problem in snowflake few months so saw this easy bugfix.

Reply:
don't have okta access. this might break. So let me just remove this.